### DISCUSSION TITLE
FIWARE, Quantum Leap, Fledge IoT, Zaphiro Technologies

### DISCUSSION THREAD

**Discussion by amotl** [crate/crate-clients-tools#197] [Ideas]
## About

> Fledge is a  leading open source industrial edge platform for industrial data pipelines, intelligent edge applications and ML.
>
> -- https://lfedge.org/projects/fledge/

## Introduction

Federico has been active at the FIWARE / Quantum Leap project, and he was a guest within one of our presentations in 2021.
- [Dr. Federico M. Facca talks at the Open Source Release Event of CrateDB v4.5](https://www.youtube.com/watch?v=fwXZtSxLCdw)

## Details

It looks like @chicco785 is now at @zaphiro-technologies, doing things with @fledge-iot technologies, no longer at FIWARE/[QuantumLeap](https://github.com/orchestracities/ngsi-timeseries-api)?

> Fledge is a  leading open source industrial edge platform for industrial data pipelines, intelligent edge applications and ML.
>
> Our Vision: Fledge’s project and community deliver data management for automation, advanced process control and IIoT, accelerating global smart manufacturing / Industry 4.0. As manufacturing becomes more data driven, the convergence of OT’s & IT’s people, processes and technology gains greater importance.
>
> Like Apache Spark’s pipelines for data lakes, Fledge’s pipelines for the automation pyramid and IIoT unify data acquisition, raw integration of data, filter/clean/transform data, aggregate data and enable edge 4.0 application development, provisioning, execution and management.

## Q & A
All of that is implemented in https://github.com/fledge-iot/fledge as a generalized FOSS CrateOM?

**Comment by amotl** [comment]
## Real-time grid monitoring

> Our grid monitoring module increases operator’s visibility of the grid state and reduces reaction time in case of hazardous situations. Measurements and alarms can be displayed in real-time into the SCADA/DMS and in a dedicated user-interface.

![image](https://github.com/crate/devrel/assets/453543/50e49655-d723-4e18-a259-3739dd8f7d51)
-- https://zaphiro.ch/solution/

**Comment by chicco785** [comment]
@amotl i confirm I am not more employed at Martel, you can contact @c0c0n3 for QuantumLeap

@fledge-iot is not using cratedb (or any timeseries db for what matters), it's a protocol gateway, so it stores in memory data for the short time needed during protocol translation.
